longevity_test.py

```diff
@@ -97,6 +97,9 @@ def test_custom_time(self):
         self.run_pre_create_keyspace()
         self.run_pre_create_schema()
 
+        if prepare_cs_user_profiles := self.params.get('prepare_cs_user_profiles'):
```

Comment:
@soyacz Lukasz, I assume you are working on `longevity_test.py`. Can you, please, check if those changes here, in my PR, are relevant? It's taken from Yaron's commit and I am not sure that it is needed

Reply:
I'm not using this approach, just basic `prepare_write_cmd` and `stress_cmd` worked for me:
https://github.com/scylladb/scylla-cluster-tests/pull/6721/files#diff-fcdf3106f25365f5cfcff985665b092bc6cc85bef47c6e0809b875226f7e1ca0

Reply:
Thanks, so I'll remove it